contracts/BIFI/infra/BeefyFeeConfigurator.sol

```diff
@@ -0,0 +1,143 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+
+interface IStrategy {
+    function vault() external view returns (address);
+}
+
+contract BeefyFeeConfigurator is OwnableUpgradeable {
+
+    struct FeeCategory {
+        uint256 total;
+        uint256 beefy;
+        uint256 call;
+        uint256 strategist;
+        string label;
+        bool active;
+    }
+
+    address public keeper;
+    uint256 public totalLimit;
+    uint256 constant DIVISOR = 1 ether;
+
+    mapping(address => uint256) public vaultFeeId;
+    mapping(uint256 => FeeCategory) internal feeCategory;
+
+    event UpdateVault(address indexed vault, uint256 indexed id);
+    event SetFeeCategory(
+        uint256 indexed id,
+        uint256 total,
+        uint256 beefy,
+        uint256 call,
+        uint256 strategist,
+        string label,
+        bool active
+    );
+    event SetTotalLimit(uint256 totalLimit);
```

Comment:
Is this an old event? I don't see where it is used.

Reply:
Old event from when the total fee could be changed, removed that function now so event is obselete